mlflow/lightgbm.py

```diff
@@ -292,6 +292,10 @@ def autolog():
         - inferred signature of the inputs and outputs of the model.
 
     Note that the `scikit-learn API`_ is not supported.
+
+    :param log_input_example: whether to log a sample of the training data
```

Comment:
Nit: Are we using the full available line length here? the newline placement seems a bit odd